### ISSUE TITLE
UV Viewer: insufficient contrast on Go button

### ISSUE THREAD

**Issue by erinchambers** [CDRH/sparc#393]
The white text and teal background of the "Go" button are just short of the minimum contrast ratio. We just need to darken the blue a bit until it meets the 4.5:1 ratio.
